Backend.ts

```diff
@@ -43,11 +43,24 @@ export class Backend<T> {
 			})
 		)
 	}
-	async list(document: { shard: string }): Promise<Document<T>[] | undefined> {
+	async list(
+		document: { shard: string },
+		limit?: number,
+		continuation?: string,
+		path?: string
+	): Promise<{ data: Document<T>[]; header: string | undefined }> {
+		path = path ? '["' + path.replace(/[^a-zA-Z0-9.]/g, "").replace(/\./g, '"]["') + '"]' : '["value"]["created"]'
 		const result: Document<T>[] = []
 		let response: cosmos.FeedResponse<Document<T> & cosmos.Document> | undefined
 		do {
-			response = response?.hasNext
+			response = limit
+				? await this.client.queryDocuments<Document<T> & cosmos.Document>({
+						maxItems: limit,
```

Comment:
Maybe we should have a fallback value? According to one article I read maxItems in cosmosDb defaults to 100, if that is true we should have a higher default.